PowerEditor/src/NppCommands.cpp

```diff
@@ -45,6 +45,7 @@ void Notepad_plus::macroPlayback(Macro macro)
 {
 	LongRunningOperation op;
 
+	_playingBackMacro = true;

```

Comment:
`macroPlayback()` is called from more than one location. When `IDM_MACRO_PLAYBACKRECORDEDMACRO` was triggered then `_playingBackMacro` was getting appropriately set so that `SCN_CHARADDED` was skipping auto-completion and auto-indentation. 

However the two other places `macroPlayback()` is called from did not set these flags. I figured it was safer to put these flags inside `macroPlayback()` instead of manually setting/unsetting them outside of it.